Sources/FoundationEssentials/FileManager/FileOperations+Enumeration.swift

```diff
@@ -10,7 +10,92 @@
 //
 //===----------------------------------------------------------------------===//
 
-#if !os(Windows)
+#if os(Windows)
+
+import WinSDK
+
+struct _Win32DirectoryContentsSequence: Sequence {
+    final class Iterator: IteratorProtocol {
+        struct Element {
+            var fileName: String
+            var fileNameWithPrefix: String
+            var dwFileAttributes: DWORD
+        }
+
+        private var hFind: HANDLE?
+        private var ffdData: WIN32_FIND_DATAW = .init()
+        private var prefix: String = ""
+        private var slash: Bool
+
+        var error: CocoaError?
+
+        init(path: String, appendSlashForDirectory: Bool, prefix: [String]) {
+            self.slash = appendSlashForDirectory
+
+            hFind = try? "\(path)\\*".withNTPathRepresentation {
```

Comment:
Should we just catch the error from here and set it to `error` so that we produce an error with the most detailed/correct info?

Reply:
Sure.